frontend/src/ui/Form.ts

```diff
@@ -265,7 +270,7 @@ const InputStyled = styled.div`
   }
 
   &:not(:last-child) .help {
-    margin-bottom: ${space(2)};
+    /* margin-bottom: ${space(2)}; */
```

Comment:
better to remove it than comment the line

Reply:
You are absolutely right.